vic/drivers/image/src/get_global_param.c

```diff
@@ -777,6 +777,9 @@ get_global_param(FILE *gp)
             else if (strcasecmp("SKIPYEAR", optstr) == 0) {
                 sscanf(cmdstr, "%*s %hu", &global_param.skipyear);
             }
+            else if (strcasecmp("COORD_DIMS_OUT", optstr) == 0) {
+                sscanf(cmdstr, "%*s %zu", &options.COORD_DIMS_OUT);
+            }
```

Comment:
> About the coordinate names: I know about those plans. Shall we implement this?

I think that makes sense, since you are already in the neighborhood.

>Do you have any suggestions about how to get this info from the domain file (how many dimensions the output of the lat/lon values should have)

`nc_inq_var` should do it.  In [this](https://www.unidata.ucar.edu/software/netcdf/docs/netcdf-c/nc_005finq_005fvar.html#nc_005finq_005fvar) example, the number of dimensions (`rh_ndims`) is determined in the second to last line.